codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/smithy/RuntimeType.kt

```diff
@@ -193,14 +193,57 @@ data class RuntimeType(val path: String, val dependency: RustDependency? = null)
      * The companion object contains commonly used RuntimeTypes
      */
     companion object {
+        /** Scope that contains all Rust prelude types, but not macros or functions */
```

Comment:
Nice! Maybe it's useful to include a link to prelude contents like [this one?](https://doc.rust-lang.org/std/prelude/index.html#prelude-contents)

Reply:
Fixed in 49d10aecb5d768cece726b2aaab87a1136852040